kclvm/error/src/lib.rs

```diff
@@ -337,9 +337,16 @@ pub enum ParseError {
     Message {
         message: String,
         span: Span,
+        fix_info: Option<FixInfo>,
     },
 }
 
+#[derive(Debug, Clone)]
+pub struct FixInfo {
+    pub error_kind: ErrorKind,
+    pub additional_info: Option<String>,
+}
+
```

Comment:
Here, I have expanded the ParseError enum like this.

Reply:
Would you please add more comments to explain this?

Reply:
For some errors like this:
https://github.com/kcl-lang/kcl/blob/26b1b17c6a94d2239153a3a59256a8c5b2ebdff7/kclvm/parser/src/parser/stmt.rs#L43-L44
additional information such as `token` or start and end position may be required for quick fix generation, that's why added an additional field here for now.